pipeline.py

```diff
@@ -252,6 +252,17 @@ def build_e2e_image(config: DevConfig) -> None:
         create_and_push_manifest(config, config.e2e_image, config.gh_run_id)
 
 
+"""
+generates docker manifests by running the following commands:
+1. Clear existing manifests
+docker manifest rm config.repo_url/image:tag
+2. Create the manifest
+docker manifest create config.repo_url/image:tag --amend config.repo_url/image:tag-amd64 --amend config.repo_url/image:tag-arm64
+3. Push the manifest
+docker manifest push config.repo_url/image:tag
+"""
+
```

Comment:
can we make it directly above the func (I also prefer above, but I believe pep python says it should be under, for me its fine either way as long as its consistent)?

Reply:
It looks like the pre-commit hook did that. Should I disable it for this?

Reply:
Oh, let me check that. Don't worry about that then

Reply:
Ah I believe its probably because that docstring is above the function which black does not recognize and therefore it adds those \n. 

Could you test adding that directly below the func and run the pre-commit again?

Reply:
Still adds whitespace even below.

Reply:
Hm, then feel free to keep it above, I will check at some point why this is the case 😓